index.js

```diff
@@ -98,6 +101,37 @@ const baseOpen = async options => {
 		}));
 	}
 
+	if (app === 'browser') {
+		const browser = await defaultBrowser();
+		const browserName = browser.name.toLowerCase();
+		return baseOpen({
+			...options,
+			app: {
+				name: open.apps[browserName],
```

Comment:
You will need to do some normalisation of the names. `browserName` will be different depending on the OS.